tests/l2/dgpu/README.md

```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
   Attached to: #89 (PCI bridge)
 ```                        
 
+### Verify Media support for Intel® Data Center GPU
+It is only supported for Intel® Data Center GPU Flex Series.
+
+NOTE: Please note that the Intel media driver (https://github.com/intel/media-driver) library is compiled while building the image and contains implementations of various Media codecs. Those codecs are patented and are owned by their respective developers.
+
+#### vainfo
+This workload runs [vainfo](https://github.com/intel/libva-utils) utilizing the i915 resource from GPU provisioning and displays the Media fetures supported by the GPU.
+
+*	Build the workload container image.
+
+```
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/tests/l2/dgpu/vainfo_build.yaml
+```
+
+*	Deploy and execute the workload.
+
+```
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/tests/l2/dgpu/vainfo_job.yaml
+```
+
+* Check the results.
+```
+  $ oc get pods -n intel-dgpu
+  NAME                        READY   STATUS      RESTARTS   AGE
+  intel-dgpu-vainfo-1-build   0/1     Completed   0          9m49s
+  intel-dgpu-vainfo-rbdvz     0/1     Completed   0          45s
+```
+```
+$ oc logs intel-dgpu-vainfo-rbdvz -n intel-dgpu
+libva info: VA-API version 1.20.0
```

Comment:
please remove line 119 to 123

Reply:
why not show the exact logs it produces?

Reply:
removed those lines.